src/wallet/wallet_rpc_server.cpp

```diff
@@ -911,6 +1094,10 @@ namespace tools
       available = false;
     }
 
+    std::set<uint32_t> subaddr_indices;
+    for (uint32_t i : req.subaddr_indices)
+      subaddr_indices.insert(i);
```

Comment:
This bit of code is repeated many times, factor it ?
Or, even better, why not hve te right type in the first place ?